stubs/3.2/typing.pyi

```diff
@@ -36,9 +36,9 @@ AnyStr = TypeVar('AnyStr', str, bytes)
 
 # Abstract base classes.
 
-_T = TypeVar('_T')
-_KT = TypeVar('_KT')
-_VT = TypeVar('_VT')
+_T = TypeVar('_T', covariant=True)
```

Comment:
Mutable collections should have invariant type variables (e.g., `MutableSequence`).